### PR TITLE
dev-util/geany-plugins: version bump to 1.35

### PR DESCRIPTION
**Pull request by cjmayo**
Signed-off-by: Chris Mayo <aklhfex@gmail.com>
Package-Manager: Portage-2.3.62, Repoman-2.3.11

---

A rename with an increase of geany dependency version to 1.35.

### CODE REVIEW

File: dev-util/geany-plugins/geany-plugins-1.35.ebuild [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+	scope? (
+		gtk3? ( x11-libs/vte:2.91 )
+		!gtk3? ( x11-libs/vte:0 )
+		)
+	soup? ( net-libs/libsoup:2.4 )
+"
+RDEPEND="${COMMON_DEPEND}
+	scope? ( sys-devel/gdb )
+"
+DEPEND="${COMMON_DEPEND}"
```
Comment (a17r) [inline dev-util/geany-plugins/geany-plugins-1.35.ebuild]: this lines tells me that `COMMON_DEPEND` really is superfluous here.

File: dev-util/geany-plugins/geany-plugins-1.35.ebuild [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+	git? ( dev-libs/libgit2:= )
+	gpg? ( app-crypt/gpgme:1= )
+	gtkspell? (
+		gtk3? ( app-text/gtkspell:3= )
+		!gtk3? ( app-text/gtkspell:2 )
+		)
+	lua? ( dev-lang/lua:0= )
+	markdown? (
+		app-text/discount
+		net-libs/webkit-gtk
```
Comment (a17r) [inline dev-util/geany-plugins/geany-plugins-1.35.ebuild]: no slot?

File: dev-util/geany-plugins/geany-plugins-1.35.ebuild [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+}
+
+src_install() {
+	default
+
+	find "${D}" -name '*.la' -delete || die
+
+	# make installs all translations if LINGUAS is empty
+	if [[ -z "${LINGUAS-x}" ]]; then
+		rm -r "${ED%/}/usr/share/locale/" || die
```
Comment (a17r) [inline dev-util/geany-plugins/geany-plugins-1.35.ebuild]: `ED` isn't slash terminated anymore in EAPI-7.

File: dev-util/geany-plugins/geany-plugins-1.35.ebuild (hunk 1 above)
Comment (cjmayo) [reply dev-util/geany-plugins/geany-plugins-1.35.ebuild]: Yes, done. Leftover from EAPI 7 update.

File: dev-util/geany-plugins/geany-plugins-1.35.ebuild (hunk 2 above)
Comment (cjmayo) [reply dev-util/geany-plugins/geany-plugins-1.35.ebuild]: Added :4.

File: dev-util/geany-plugins/geany-plugins-1.35.ebuild (hunk 3 above)
Comment (cjmayo) [reply dev-util/geany-plugins/geany-plugins-1.35.ebuild]: Removed.

**Pull request closed without merging**